.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!-- This comments are hidden when you submit the issue, so you do not need to remove them! -->
+<!-- Please be sure to check out our [contributing guidelines](https://github.com/sunpy/sunpy/blob/master/CONTRIBUTING.rst). -->
+<!-- Please be sure to check out our [code of conduct](https://github.com/sunpy/sunpy/blob/master/CODE_OF_CONDUCT.rst). -->
+
+<!-- Please just have a quick search on GitHub to see if a similar issue has already been posted. -->
+<!-- If a similar issue is closed, have a quick look to see if you are satisfied by the resolution. -->
+<!-- If not please go ahead and open an issue! -->
```

Comment:
Or reopen that one??

Reply:
They probably can't without more repo access.